xml/docu_styleguide.structure.xml

```diff
@@ -2535,6 +2535,74 @@ xml:id="tab-install-source"</screen>
 <screen>&lt;phrase role="productname"&gt;LibreOffice&lt;/phrase&gt; is an office suite.</screen>
  </sect2>
 
+ <sect2 xml:id="sec-profiling">
+  <title>Profiling</title>
+  <para>
+   Profiling is convenient for the creation of consistent documentation 
+   across different products or product lines. This is especially beneficial 
+   when similar products share a considerable amount of features, with only 
+   minor differences. Instead of maintaining separate documentation for
```

Comment:
I think "minor" is unfortunate. Maybe it's just me, but it gives me the impression that you have only a tiny amount of changes that are different.
The reality is, it works also for more changes.

Long story short: I'd suggest to use "few":

```suggestion
   few differences. Instead of maintaining separate documentation for
```